Description: fixed a typo in option parsing code which made -D always
             random when random -S was used. (Closes: #298799)
Author: Dennis Vshivkov <walrus@amur.ru>
Last-Update: 2005-03-10
Index: packit-1.0/src/main.c
===================================================================
--- packit-1.0.orig/src/main.c
+++ packit-1.0/src/main.c
@@ -473,7 +473,7 @@ parse_inject:
 
             case 'S':
                 if(strlen(optarg) == 1 && !strncmp(optarg, "R", 1))
-                    rand_d_port = 1;
+                    rand_s_port = 1;
                 else
                     rand_s_port = 0;
 
